src/components/form-helpers/form-helpers.js

```diff
@@ -39,6 +39,15 @@ function translateIntlTelInputCountryNames(language) {
           : nameInCurrentLanguage;
     }
   }
+  countryData.sort((a, b) => {
+    if (a.name.toLocaleLowerCase() < b.name.toLocaleLowerCase()) {
```

Comment:
What do you think about using [this approach](https://stackoverflow.com/questions/8996963/how-to-perform-case-insensitive-sorting-in-javascript#49004987)?

```
[ 'Foo', 'bar' ].sort((a, b) => a.localeCompare(b, undefined, { sensitivity: 'base' }));
```

Maybe some browsers do not support it, but in that case, it is not so bad.